docs/nightly/en/getting-started/quick-start.md

```diff
@@ -0,0 +1,368 @@
+# Quick Start
+
+Before proceeding, please ensure you have [installed GreptimeDB](./installation/overview.md).
+
+This guide will walk you through creating a metric table and a log table, highlighting the core features of GreptimeDB.
+
+## Connect to GreptimeDB
+
+If your GreptimeDB instance is running on `127.0.0.1` with the MySQL client default port `4002` or the PostgreSQL client default port `4003`,
+you can connect to it using the following commands:
+
+```shell
+mysql -h 127.0.0.1 -P 4002
+```
+
+Or
+
+```shell
+psql -h 127.0.0.1 -p 4003 -d public
+```
+
```

Comment:
Do we need to add a note that GreptimeDB supports many protocols, but for quick start, we are using SQL for the demo?

Reply:
have added a line to note mutiple protocols.